packages/contracts-bedrock/test/properties/PROPERTIES.md

```diff
@@ -0,0 +1,72 @@
+# supertoken properties
+
+legend:
+- `[ ]`: property not yet tested
+- `**[ ]**`: property not yet tested, dev/research team has asked for extra focus on it
+- `[X]`: tested/proven property
+- `[~]`: partially tested/proven property
+- `:(`: property won't be tested due to some limitation
+
+## Unit test
+
+| id  | description                                                                        | halmos | medusa |
+| --- | ---                                                                                | ---    | ---    |
+| 0   | supertoken token address does not depend on the executing chain’s chainID          | [ ]    | [ ]    |
+| 1   | supertoken token address depends on name, remote token, address and decimals       | [ ]    | [ ]    |
+| 2   | convert() should only allow converting legacy tokens to supertoken and viceversa   | [ ]    | [ ]    |
```

Comment:
allowed legacy and allowed supertokens

Reply:
not every legacy nor supertoken will be allowed

Reply:
this was defined in `SUMMARY.md`, however I realize it's not really clear from this document, so I merged them both and made sure the definitions are above